server/service/individualServices/ReadInventoryData.js

```diff
@@ -7,97 +7,268 @@ const FIRMWARE = {
+      pathParams.push(mountName);
+      pathParams.push(equipmentUuid);
+      let consequentOperationClientAndFieldParams = await IndividualServiceUtility.getConsequentOperationClientAndFieldParams(forwardingName, stringName)
+      _traceIndicatorIncrementer = ++_traceIndicatorIncrementer;
+      let equipmentInfoResponse = await IndividualServiceUtility.forwardRequest(consequentOperationClientAndFieldParams, pathParams, requestHeaders, _traceIndicatorIncrementer);
+      if (Object.keys(equipmentInfoResponse).length === 0) {
+        console.log(`${forwardingName} is not success`);
+        return new createHttpError.InternalServerError();
+      }
+      equipmentInfoList.push(equipmentInfoResponse);
+    }
+
+    /****************************************************************************************************
+      * Formulate equipmentInfoResponse from eqipmentInfoList
+    *****************************************************************************************************/
+
+    let equipmentInfo = await formulateEquipmentInfo(equipmentInfoList);
+    equipmentInfoResponse.traceIndicatorIncrementer = _traceIndicatorIncrementer;
+    equipmentInfoResponse.equipmentInfo = equipmentInfo;
+
+    return equipmentInfoResponse;
+  } catch (error) {
+    return new createHttpError.InternalServerError(`${forwardingName} is not success with ${error}`);
+  }
+}
+
+/**
+* Formulate equipment info from equipmentInfoList
+* @param {list} equipmentInfoList List of equipment information
+* @returns {Object} return classified equipment info 
+*/
+async function formulateEquipmentInfo(equipmentInfoList) {
+  let equipmentInfo = {};
+  let equipment = {};
+  for (let i = 0; i < equipmentInfoList.length; i++) {
+    let manufacturedThing = equipmentInfoList[i][EQUIPMENT.CORE_MODEL_ACTUAL_EQUIPMENT][EQUIPMENT.ACTUAL_EQUIPMENT.MANUFACTURED_THING];
+    let category = equipmentInfoList[i][EQUIPMENT.CORE_MODEL_ACTUAL_EQUIPMENT][EQUIPMENT.ACTUAL_EQUIPMENT.STRUCTURE][EQUIPMENT.ACTUAL_EQUIPMENT.CATEGORY];
+    equipment[EQUIPMENT.ACTUAL_EQUIPMENT.EQUIPMENT_NAME] = manufacturedThing[EQUIPMENT.ACTUAL_EQUIPMENT.EQUIPMENT_TYPE][EQUIPMENT.ACTUAL_EQUIPMENT.TYPE_NAME];
+    equipment[EQUIPMENT.ACTUAL_EQUIPMENT.SERIAL_NUMBER] = manufacturedThing[EQUIPMENT.ACTUAL_EQUIPMENT.EQUIPMENT_INSTANCE][EQUIPMENT.ACTUAL_EQUIPMENT.SERIAL_NUMBER];
+    equipment[EQUIPMENT.ACTUAL_EQUIPMENT.PART_NUMBER] = manufacturedThing[EQUIPMENT.ACTUAL_EQUIPMENT.EQUIPMENT_TYPE][EQUIPMENT.ACTUAL_EQUIPMENT.PART_TYPE_IDENTIFIER];
+    if (category === EQUIPMENT.EQUIPMENT_CATEGORY.MODULE + ":" + EQUIPMENT.EQUIPMENT_CATEGORY.MODEM) {
+      equipmentInfo.modem = equipment;
+    } else if (category === EQUIPMENT.EQUIPMENT_CATEGORY.MODULE + ":" + EQUIPMENT.EQUIPMENT_CATEGORY.FULL_OUTDOOR_UNIT) {
+      equipmentInfo.radio = equipment;
+    } else if (category === EQUIPMENT.EQUIPMENT_CATEGORY.MODULE + ":" + EQUIPMENT.EQUIPMENT_CATEGORY.OUTDOOR_UNIT) {
+      equipmentInfo.device = equipment;
```

Comment:
As per the specification , device is FULL_OUTDOOR_UNIT. Please change condition from OUTDOOR_UNIT to FULL_OUTDOOR_UNIT.
![image](https://github.com/openBackhaul/AutomatedLinkAcceptanceProxy/assets/62344562/3e9c330e-a458-4ee8-9398-d3b165c56d25)

Reply:
Thank you for the comment . Sorry , for the copy paste mistake while removing the hardcoding of variables.Changed it now.

Reply:
Thank you for the comment . Sorry , for the copy paste mistake while removing the hardcoding of variables.Changed it now.